spec/rails_helper.rb

```diff
@@ -72,6 +72,18 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+  config.before(:suite) do
+    # we include this because at present (2020-05-06) Vita Partners
+    # are yaml-loaded read-only domain information, as are states.
+    # TODO: when Vita Partners are editable, this should be removed
+    # and fixed tests adjusted
+    extend StateImporter
```

Comment:
i guess Tom's above comment about doing this instead of `StateImporter.upsert_states` also applies here